tests/contrib/storage/test_linux_ceph.py

```diff
@@ -584,6 +584,8 @@ def test_erasure_pool_create(self, get_osds, erasure_profile):
             call(['ceph', '--id', 'admin', 'osd', 'pool', 'create',
                   '--pg-num-min=32', 'test',
                   '2048', '2048', 'erasure', 'default']),
+            call(['ceph', '--id', 'admin', 'osd', 'pool',
+                  'set', 'test', 'target_size_ratio', '1.0']),
```

Comment:
Note that this was called after creation of erasure coded pools before the change too, but the order has now changed which means we need to include it in the check.